standalone/lib/annotator.js

```diff
@@ -119,7 +119,7 @@ function bindHanlers() {
     /* Binds handlers to DOM elements. */
 
     // TODO: causes errors if called before the cy is initialised
-    $(document).keyup(keyUpClassifier);
+    $(document).keydown(keyUpClassifier);
```

Comment:
Shouldn't this be `keyDownClassifier` if it's on a `.keydown()` event?

Reply:
Right, forgot to change the name :)